index.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <!-- Google Fonts -->
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Baloo+2&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Fontawesome -->
+    <script
+      src="https://kit.fontawesome.com/9469188226.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <!-- Bootstrap CSS -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+
+    <title>Homepage</title>
+  </head>
+
+  <body>
+    <!-- Nav Bar -->
+    <nav class="navbar navbar-expand-lg navbar-dark fixed-top">
+      <a class="navbar-brand" href="index.html">Kizomba ATX</a>
+      <button
+        class="navbar-toggler"
+        type="button"
+        data-toggle="collapse"
+        data-target="#navbarNav"
+        aria-controls="navbarNav"
+        aria-expanded="false"
+        aria-label="Toggle navigation"
+      >
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarNav">
+        <ul class="navbar-nav">
+          <li class="nav-item active px-3">
+            <a class="nav-link" href="index.html"
+              >Home <span class="sr-only">(current)</span></a
+            >
+          </li>
+          <li class="nav-item px-3">
+            <a class="nav-link" href="kizGetdown.html">KizGetdown</a>
+          </li>
+          <li class="nav-item px-3">
+            <a class="nav-link" href="classes.html">Classes</a>
+          </li>
+          <li class="nav-item px-3">
+            <a class="nav-link" href="instagram.html">Gram</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+
+    <!-- Header-->
+    <header class="header">
+      <div class="overlay">
+        <div class="title-position">
+          <h1 class="hero-title">KIZOMBA COMMUNITY OF AUSTIN,TX</h1>
```

Comment:
Open the DevTools and simulate the mobile view (iPhone 5 or 6), [see how to do it](https://developers.google.com/web/tools/chrome-devtools/device-mode) you'll notice the title is too big. It needs to adapt better, otherwise the page is broken.